common/subscriber.go

```
@@ -55,6 +56,16 @@ func (self *IdentityService) IsEncryptionRequired() bool {
 	return self.Service.EncryptionRequired
 }
 
+func (self *IdentityService) GetConfig(configTypeName string, v any) (bool, error) {
+	if config, ok := self.Configs[configTypeName]; ok {
+		if err := json.Unmarshal([]byte(config.Config.DataJson), &v); err != nil {
+			return false, err
+		}
+		return true, nil
+	}
+	return false, nil
```

feels like should return an error here